.vib/nginx-ingress-controller/goss/goss.yaml

```diff
@@ -0,0 +1,20 @@
+addr:
+  tcp://nginx-ingress-controller:{{ .Vars.service.ports.https }}:
+    reachable: true
```

Comment:
Is there any reason you use `addr` over `http` (given that this is an HTTP port)?

Reply:
None that I remember to be honest. Changed after testing this using `http` without issues.